src/style.css

```diff
@@ -1,3 +1,105 @@
+/* @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap'); */
+
 body {
   margin: 0;
+  --header-height: 60px;
+  /* background-color: gainsboro; */
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+nav {
+  padding: 0;
+  margin: 0 51px 0 0;
+}
+
+.nav__list.nav__list:not(:last-child) {
+  margin-right: 20px;
+}
+
+@media (min-width: 1024px) and (max-width: 1199px) {
+
+  nav {
+    padding: 0;
+    margin: 0 51px 0 0;
+  }
+
+  .nav__list.nav__list:not(:last-child) {
+    margin-right: 20px;
+  }
+
+}
+
+.header {
+  margin: 0 auto;
+  background-color: white;
+  display: flex;
+  /* height: var(--header-height); */
```

Comment:
and align-items: center rule and remove margins from logo

Reply:
when it changed the auto test stopped working